src/main/java/Duke.java

```diff
@@ -1,10 +1,237 @@
+
+        scrollPane.setVvalue(1.0);
+        scrollPane.setFitToWidth(true);
+
+        // You will need to import `javafx.scene.layout.Region` for this.
+        dialogContainer.setPrefHeight(  Region.USE_COMPUTED_SIZE);
+
+        userInput.setPrefWidth(325.0);
+
+        sendButton.setPrefWidth(55.0);
+
+        AnchorPane.setTopAnchor(scrollPane, 1.0);
+
+        AnchorPane.setBottomAnchor(sendButton, 1.0);
+        AnchorPane.setRightAnchor(sendButton, 1.0);
+
+        AnchorPane.setLeftAnchor(userInput , 1.0);
+        AnchorPane.setBottomAnchor(userInput, 1.0);
+
+        // more code to be added here later
+        //Step 3. Add functionality to handle user input.
+        sendButton.setOnMouseClicked((event) -> {
+            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+            userInput.clear();
+        });
+
+        userInput.setOnAction((event) -> {
+            dialogContainer.getChildren().add(getDialogLabel(userInput.getText()));
+            userInput.clear();
+        });
+
+        dialogContainer.heightProperty().addListener((observable) -> scrollPane.setVvalue(1.0));
+    }
+
+    /**
+     * Iteration 1:
+     * Creates a label with the specified text and adds it to the dialog container.
+     * @param text String containing text to add
+     * @return a label with the specified text that has word wrap enabled.
+     */
+    private Label getDialogLabel(String text) {
+        // You will need to import `javafx.scene.control.Label`.
+        Label textToAdd = new Label(text);
+        textToAdd.setWrapText(true);
+
+        return textToAdd;
+    }
+
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
+    String getResponse(String input, String[] parts) throws InsufficientArgumentsException, FileNotFoundException {
+        int listCounter = 1;
+        switch (input) {
+        case "done": {
+            int index = Integer.parseInt(parts[1]);
+            Task toMark = tasks.get(index - 1);
+            toMark.markAsDone();
+            TaskList.getTasklist().get(index - 1).markAsDone();
+            storage.save();
+            return toMark.toString();
+        }
+        case "todo":
+        case "deadline":
+        case "event":
+            if (parts.length == 1) {
+                throw new InsufficientArgumentsException("Insufficient arguments provided");
+            }
+            StringBuilder str = new StringBuilder();
+            if (input.equals("todo")) {
+                for (int i = 1; i < parts.length; i++) {
+                    str.append(" ");
+                    str.append(parts[i]);
+                }
+                String taskString = str.toString();
+                Todo todo = new Todo(taskString);
+                tasks.add(todo);
+                storage.save();
+                return todo.toString();
+            } else {
```

Comment:
Could have line breaks in between to make it more readable for others. More info [here](https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically).

Reply:
Noted. Thank you for the feedback.